### DISCUSSION TITLE
Projects Beta - No link to repo from Kanban board??

### DISCUSSION THREAD

**Discussion by taylor-lindores-reeves** [community/community#16313] [Projects and Issues]
Why is there nowhere to get to the project repository from the project dashboard (Kanban / Sprint) section?

**Comment by OneCyrus** [comment]
switching between repo and board is a bit painfull. it would be great to have a more streamline integration in the UI.

**Comment by borekb** [comment]
Maybe this could be considered to be part of "projects parity" – https://github.com/github/roadmap/issues/483. Navigation between a board and the rest of the repo is currently much smoother with Projects Classic. (cc @ankneis)
